src/porklock/commands.clj

```diff
@@ -242,6 +242,16 @@
     (if @error?
       (throw (Exception. "An error occurred tranferring files into iRODS. Please check the above logs for more information.")))))
 
+(defn- parse-source-list
+  "Returns a list of paths read from the the given `source-list` path list file, or nil if the file does not exist.
+   Assumes the first line of the path list is a header-line, and discards it from the results."
+  [source-list]
+  (when (ft/exists? (str source-list))
+    (->> source-list
+         slurp
+         string/split-lines
+         rest)))
```

Comment:
I'm picking nits here, so there's no real reason to make this change. When parsing files such as these, I tend to like to filter out blank lines and lines that begin with a hash symbol in case someone hand-writes them and assumes that comment lines and blank lines are ignored. Of course, the chances of that happening in _this_ case are almost nil, so feel free to ignore this comment. 😄 

_Edit:_ Heh, and I see from the code below that blank lines (but not comments) will be ignored; it just happens below instead. My first thought was that I'd rather have the filtering done here, but I see why it was placed below. In other words, carry on. 😄

Reply:
Ah, good point about filtering out comments. I hadn't thought of that 🤔 

But yeah, I'm also filtering out empty `--source` paths at the same time as blank lines from the parsed file for now.

Reply:
I thought this was a good idea, so I pushed 1 more commit to ignore all comment lines 👍